### ISSUE TITLE
reports

### ISSUE THREAD

**Issue by JamieYnonan** [boxed/mutmut#83]
Hi, I'm running the following:
```
pytest
mutmut run --use-coverage
mutmut junitxml
```
```
[mutmut]
paths_to_mutate=activity/
backup=False
runner=pytest -x
tests_dir=tests/

[tool:pytest]
addopts = -ra -q --junitxml=build/reports/junit/junit.xml --cov-append --cov=activity --cov-report html:build/reports/coverage/html --cov-report xml:build/reports/coverage/xml/coverage.xml
console_output_style = classic
junit_family = xunit2
junit_suite_name = activity
testpaths = tests
```

```
mutmut==1.3.1
pytest==4.2.0
pytest-cov==2.6.1
junit-xml==1.8
```

Can I save the result in some xml indicating the path?
How can I see the results (in the best way) in order to solve the problems (mutants)?

**Comment by boxed** [comment]
First an aside: It looks to me like you're running with coverage turned on when testing every mutant. This will only make everything take much longer for no benefit.

The best way to actually work with the tests it not to use the junitxml feature or report formats, but follow the instructions in the documentation. That is to:

1. run `mutmut run` (optionally with `--coverage` if you don't have 100% coverage and don't want to fix that first),
2. `mutmut results` to show the mutants
3. `mutmut apply <id>` to apply a mutant or `mutmut show <id>` to show the mutant
4. write a test that fails on the mutant
5. If you used the apply method, revert the change and see that it now passes or if you used the show method, run `mutmut run <id>` to validate your fix

And then repeat steps 2 to 5 until you're exhausted :P

(I've updated some of the readme just now, because I spotted an error while writing this. Thanks!)
